test/payload.test.js

```diff
@@ -0,0 +1,77 @@
+const tap = require('tap');
+const startServer = require('../');
+
+process.env.ENDPOINT = '/payload';
+
+tap.test('gathers string configurations correctly', async (t) => {
+  const rapptor = await startServer();
+
+  rapptor.server.settings.app.hosts = {
+    apple: 'apple'
```

Comment:
we shouldn't be supporting strings like this - think about building this as a generic service - not something tied to autodeploy